src/Components/ebikes/BikeCard.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { NavLink, useNavigate } from 'react-router-dom';
+import {
+  BiLogoTwitter, BiLogoFacebook, BiLogoInstagram,
+} from 'react-icons/bi';
+import { setEbike } from '../../redux/ebike/ebikeSlice';
+
+function BikeCard({ bike }) {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const clickAbike = () => {
+    dispatch(setEbike(bike));
+    // console.log('bike from the card', bike);
```

Comment:
- [ ] Kindly, remove the commented code found in this file. 👍

Reply:
✔️ DONE